tests/fixed/NamingCamelCase.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Example;
+
+class NamingCamelCase
+{
+    /** @var mixed */
+    public $A;
+
+    /** @var mixed */
+    protected $B;
+
+    /** @var mixed */
+    private $C;
+
+    public function fcn(string $A) : void
+    {
+        $Test = $A;
```

Comment:
These variables aren't camel cased, right?

Reply:
There is no autofixer for this sniff. So I hope I got it right here and the "fixed" fixture has no changes.

Reply:
Cool, my bad, I didn't notice it was only a sniff 👍